dao/reminders.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
           reminder.due,
           family
       ))
-      .then(result => result.lastId);
+      .then(result => this.show(family, result.lastId));
```

Comment:
I'm quite sure we shouldn't do this here in the DAO but in the higher level :)
(same for all other methods...)